R/filter_hierarchical.R

```diff
@@ -4,38 +4,58 @@
 #'
 #' This function is used to filter hierarchical table rows. Filters are not applied to summary or overall rows.
 #'
-#' @param x (`tbl_hierarchical`, `tbl_hierarchical_count`)\cr
-#'   A hierarchical gtsummary table of class `'tbl_hierarchical'` or `'tbl_hierarchical_count'`.
+#' @param x (`tbl_hierarchical`, `tbl_hierarchical_count`, `tbl_ard_hierarchical`)\cr
+#'   A hierarchical gtsummary table of class `'tbl_hierarchical'`, `'tbl_hierarchical_count'`,
+#'   or `'tbl_ard_hierarchical'`.
 #' @param filter (`expression`)\cr
 #'   An expression that is used to filter rows of the table. See the Details section below.
+#' @param var ([`tidy-select`][dplyr::dplyr_tidy_select])\cr
+#'   Hierarchy variable from `x` to perform filtering on. If `NULL`, the last hierarchy variable
+#'   from `x` (`dplyr::last(attributes(x)$args$variables)`) will be used.
```

Comment:
I think this is only accurate for the ARD case, right? `dplyr::last(attributes(x)$args$variables)`

We'd want `dplyr::last(x$inputs$variables)`?

Reply:
Thanks for spotting this!

Reply:
Oh wait, I think this needs to be `dplyr::last(x$inputs$include)`, right? If so, can you update that, then the NEWS, then merge? Thank you!!

Reply:
Those two values are always the same since the last var in `variables` has to be included in `include`, but I'll update to specify and then merge :)